Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Use an official Python runtime as a parent image
+FROM python:3.8
+MAINTAINER label="Rohan Rustagi"
```

Comment:
LABEL maintainer="Rohan Rustagi" would be the modern way to write this

Reply:
Thanks for the feedback.